submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,33 @@
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 342. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 1; 
```

Comment:
The answer is `342` but this doesn't show that is the query that made you obtain that result. There is a way query to help get that value i.e `342`